### PR TITLE
Proposal for static theme wizard

### PR DESCRIPTION
**Pull request by bobsilverberg**
This is a draft of a proposal for a GSOC project. I am going to solicit feedback from my team before requesting a merge.

### CODE REVIEW

**Comment by bobsilverberg** [comment]
@jvillalobos and/or @muffinresearch Could you please take a look and add comments if you think any changes are needed?

**Review by jvillalobos — CHANGES_REQUESTED** [review]
I think it'd be good to link to https://extensionworkshop.com/documentation/themes/, to provide some context. Also, the theme is generated using SVG, so maybe some knowledge on that is also needed?

File: proposals/static-theme-wizard.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# AMO Devhub Static Theme Wizard
+
+**Mentor:** Andrew Williamson
```
Comment (muffinresearch) [inline proposals/static-theme-wizard.md]: Presumably Andrew is onboard with the commitment for mentoring a GSOC project? Or is this intended more of a joint effort between you and Andrew if it's picked?

**Comment by bobsilverberg** [comment]
> I think it'd be good to link to [extensionworkshop.com/documentation/themes](https://extensionworkshop.com/documentation/themes/), to provide some context. Also, the theme is generated using SVG, so maybe some knowledge on that is also needed?

Good suggestions, thanks @jvillalobos.

File: proposals/static-theme-wizard.md (hunk 1 above)
Comment (bobsilverberg) [reply proposals/static-theme-wizard.md]: Good point. No, I didn't speak with Andrew about this. I'll change this to me for now. These are all just drafts anyway, and if Mozilla is accepted into the program we can change these proposals later on.

**Pull request merged**